src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
+  }
+  return price;
```

Comment:
The function name `calculateRentalCost` is descriptive, which is good. However, the use of literals in the expressions does not follow the best practices as per the checklist. Use constants to represent the values such as the daily cost, long term rental days, and discounts to make the code more readable and maintainable.